src/components/DrowsyStatsModal.jsx

```
@@ -0,0 +1,63 @@
+
+        <div className="text-center">
+          <button
+            onClick={() => {
+              onConfirm();
+              onClose();
+            }}
+            className="bg-cornflower-400 hover:bg-cornflower-500 rounded-xl px-6 py-2 text-white transition"
+          >
+            확인
+          </button>
+        </div>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)